content/SCALE/SCALETutorials/Credentials/ManageLocalUsersSCALE.md

```diff
@@ -46,90 +46,91 @@ Enter the name you want to use for the administrator account.
 
-{{< trueimage src="/images/SCALE/22.12/AddUserHomeDirPermSCALE.png" alt="Add User Home Directory" id="5: Add User Home Directory" >}}
+{{< trueimage src="/images/SCALE/23.10/AddUserHomeDirPermSCALE.png" alt="Add User Home Directory" id="5: Add User Home Directory" >}}
 
 When creating a user, the home directory path is set to <file>/nonexistent</file>, which does not create a home directory for the user.
 To set a user home directory, enter a path in **Home Directory** or select it using the file browser.
-If the directory exists and matches the user name, TrueNAS sets it as the user home directory.
-When the path does not end with a sub-directory matching the user name, TrueNAS creates a new sub-directory if the **Create Home Directory** checkbox is enabled.
+If the directory exists and matches the username, TrueNAS sets it as the user home directory.
+When the path does not end with a sub-directory matching the username, TrueNAS creates a new sub-directory if the **Create Home Directory** checkbox is enabled.
 TrueNAS shows the path to the user home directory when editing a user.
 
-You can set the home directory permissions directly under the file browser. 
-You cannot change TrueNAS default user account permissions.
+Select the **Home Directory Permissions** (**Read**, **Write**, **Execute**) for each (**User**, **Group**, **Other**) to set Unix permissions for the user home directory. Built-in users are read-only and can not modify these permissions settings.
 
 ### Configuring Authentication Settings
 
-{{< trueimage src="/images/SCALE/22.12/AddUserHomeDirAuthSCALE.png" alt="Add User Home Directory and Authentication Settings" id="6: Add User Home Directory and Authentication Settings" >}}
+{{< trueimage src="/images/SCALE/23.10/AddUserHomeDirAuthSCALE.png" alt="Add User Home Directory and Authentication Settings" id="6: Add User Home Directory and Authentication Settings" >}}
 
 You can assign a public SSH key to a user for key-based authentication by entering or pasting the *public* key into the **Authorized Keys** field.
 
 {{< hint type=important >}}
 Do *not* paste the private key.
 {{< /hint >}}
 
+You can also click **Choose File** under **Upload SSH Key** and browse to the location of an SSH key file.
+
 If you are using an SSH public key, always keep a backup of the key.
 
-Select the [shell]({{< relref "LocalUsersScreensSCALE.md" >}}) option for the user from the **Shell** dropdown options. 
-Set the local admin user shell to **TrueNAS CLI** to open shell in the TrueNAS CLI. Set to **TrueNAS Console** to open in the Console Setup Menu for SCALE. 
+Select the [shell]({{< relref "LocalUsersScreensSCALE.md" >}}) option for the user from the **Shell** dropdown options.
+Options are **nologin**, **bash**, **rbash**, **dash**, **sh**, **tmux**, and **zsh**.
+Admin users can also use **TrueNAS CLI** to open shell in the TrueNAS CLI and **TrueNAS Console** to open in the Console Setup Menu for SCALE.
```

Comment:
so maybe just describe these new options on the list this way:
```suggestion
Select **TrueNAS CLI** to open shell in the TrueNAS CLI and **TrueNAS Console** to open in the Console Setup Menu for SCALE.
```

Reply:
I want to specify that you'll only see these options if for admin or root users though. I'll think about it.